src/event_form/components/calendar/form-event-calendar.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="calendar-type-picker">
     <div class="calendar-type-options">
 
-        <a href="#" ng-click="calendar.setType('single')" ng-class="{'selected': calendar.type === 'single'}">
+        <a href="#" ng-click="calendar.setType('single')" ng-class="{'selected': calendar.type === 'single' || calendar.type === 'multiple'}">
```

Comment:
De conditie in `ng-class` kan je vervangen met `!calendar.weeklyRecurring` zoals iets later in de code.